installer/Templates/Head.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Installer\Templates;
+
+use Template;
+
+class Head implements Template
+{
+	/**
+	 * @var array list of the steps (ordered)
+	 */
+	private $steps = ['Welcome', 'Requirements', 'Permissions', 'Env', 'Migrate'];
+
+	private function get_index($step)
+	{
+		return array_search($step, $this->steps);
+	}
+
+	public function print(array $input = [])
+	{
+		echo '<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
I think if a user needs this when *installing* Lychee then all hope is already lost. This should only be an issue pre IE8.

Reply:
So what do I do there ? :sweat_smile:

Reply:
I'd remove the `<meta http-equiv...>` line, but it's not going to hurt really. I may have done this review shortly after reading a summary of the mess that is user agents...